specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -2019,6 +2124,41 @@
         "query": {
           "type": "object",
           "description": "A query to retrieve data from source. Type: string (or Expression with resultType string)."
+        },
+        "partitionOption": {
+          "description": "The partition mechanism that will be used for Netezza read in parallel.",
+          "type": "string",
+          "enum": [
+            "None",
+            "DataSlice",
+            "DynamicRange"
+          ],
+          "x-ms-enum": {
+            "name": "NetezzaPartitionOption",
+            "modelAsString": true
+          }
+        },
+        "partitionSettings": {
+          "description": "The settings that will be leveraged for Netezza source partitioning.",
+          "$ref": "#/definitions/NetezzaPartitionSettings"
```

Comment:
Actually, I have a question about these changes. I cannot find the usage for the `NetezzaSource` and other definitions. Why do we do these changes?

Reply:
Hi Myron, thanks for your comments, while I am not sure I understood your concerns quite clearly here.  are you concerning on my change on Netezza only or all related new sources I am trying to add?